include/opt-sched/Scheduler/aco.h

```diff
@@ -37,6 +43,9 @@ class ACOScheduler : public ConstrainedScheduler {
   pheremone_t &Pheremone(SchedInstruction *from, SchedInstruction *to);
   pheremone_t &Pheremone(InstCount from, InstCount to);
   double Score(SchedInstruction *from, Choice choice);
+  FUNC_RESULT performSchedPass(InstSchedule *schedule_out);
+  bool shouldReplaceSchedule(ACOPass pass, InstSchedule* oldSched, InstSchedule* newSched);
+
```

Comment:
Extra whitespace

Reply:
I tried to match the LLVM style without using clang-format.  I will run it through clang-format and will do so in the future.